source/manuals/accessibility.html.md.erb

```diff
@@ -0,0 +1,115 @@
+
+### Browser compatibility testing
+
+Services should be thoroughly tested in all browsers they are required to be compatible with, including mobile devices and accessibility tools such as screen readers. Emulated browser testing services are available, but some testing on real mobile devices is also recommended.
+
+For reference, these are the [browsers that GOV.UK is required to be compatible with](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices).
+
+### Test content served by third party systems
+
+If a service is built using an existing or third party system, such as a content management system or JavaScript framework, it should be tested for accessibility. Accessibility compliance cannot be guaranteed by any system, as even small changes can introduce accessibility barriers.
+
+### Testing with real users
+
+Services should be tested by real users to evaluate how accessible they are. This can be with members of the public as well as specialist organisations who provide accessibility testing services.
+
+Services can also be tested using an [empathy lab](https://gds.blog.gov.uk/2018/06/20/creating-the-uk-governments-accessibility-empathy-lab/), such as the one at GDS, which is designed to provide a simulation of using the web with accessibility issues so developers can better understand what this is like.
+
+Testing with real users can be a time consuming process. It is recommended that this is done only after other accessibility testing has occurred first.
+
+### Consider using the GOV.UK Design System
+
+The [GOV.UK Design System](https://design-system.service.gov.uk/) is a collection of website styles, components and patterns designed to be used to build government services. It provides pre-built website markup that has been developed specifically to be accessible. Using it will also save development time and provide a look and feel consistent with other government services.
+
+While it may not have a full set of components for every service, it provides a solid foundation to work from that can be expanded to suit a services individual needs.
+
+## Specific guidance
+
+There is a common misconception that accessibility is difficult, time consuming and costly. This is not true, however the range of possible issues that could occur is too wide to detail here. Having said that, here are some common accessibility issues many websites have, why they’re a problem and how to avoid them.
+
+### Non semantic HTML
+
+Semantic HTML means using the appropriate tag for the element being described. A common example of this is not using a button element for a button, but relying on JavaScript to provide the correct action when the element is used.
+
+This is a problem because non-semantic HTML is difficult for technologies such as screen readers to understand, which means that some users will not be able to operate or interact with these elements, potentially rendering a service unusable.
```

Comment:
Maybe break this sentence up a bit?

```suggestion
This is a problem because non-semantic HTML is difficult for assistive technologies - such as screen readers - to understand. This means that some users will not be able to operate or interact with these elements, potentially rendering a service unusable.
```